py-doggie-daycare/README.md

```diff
@@ -0,0 +1,5 @@
+# Doggie Daycare 
+
+The doggie daycare sample application is a FastAPI app. The application has one endpoint to register a pet and its owner and a second endpoint to retrieve a list of registered pets for the supplied owner as shown below.
+
+This project repository is a reference for the [Your First Klotho App (Python)](https://klo.dev/docs/tutorials/your_first_klotho_app_python) tutorial.  Instructions for building, testing locally, compiling, and deploying, can be found in the tutorial.
```

Comment:
nit: I assume we just want all the info in tutorial rather than duplicating?

Reply:
That was the idea when putting this together initially.